File: LeetCode/Simulation/Python/Leetcode1252-cells-with-odd-values-in-a-matri.py
class Solution:
    def oddCells(self, m: int, n: int, indices: List[List[int]]) -> int:
        row,col = [0] * m,[0] * n
        for r,c in indices:
            row[r] += 1
            col[c] += 1

        cnt = 0
        for i in range(m):
            for j in range(n):  
                if (row[i] + col[j]) & 1: cnt += 1
        return cnt

File: LeetCode/Simulation/Golang/Leetcode1252-cells-with-odd-values-in-a-matri.go
package sbgo

func oddCells(m int, n int, indices [][]int) int {
	row, col := make([]int, m), make([]int, n)
	for _, pos := range indices {
		r, c := pos[0], pos[1]
		row[r]++
		col[c]++
	}
	cnt := 0
	for i := 0; i < m; i++ {
		for j := 0; j < n; j++ {
			if (row[i]+col[j])&1 == 1 {
				cnt++
			}
		}
	}
	return cnt
}
